src/platform/windows.rs

```diff
@@ -117,6 +118,9 @@ pub trait WindowExtWindows {
   /// Whether to show the window icon in the taskbar or not.
   fn set_skip_taskbar(&self, skip: bool);
 
+  /// Sets the taskbar progress state./
+  fn set_progress_bar(&self, progress: ProgressBarState);
```

Comment:
same as above, remove and use a unified method in `src/window.rs`

Reply:
Adding in next commit